demo.slim

```diff
@@ -7,8 +7,6 @@ html
         |Slimish-Jinja Example
     / Self closing tag with attributes.
     meta name="keywords" content="template language"
-    - assets filters='jsmin', output='gen/packed.js', 'common/jquery.js', 'site/base.js' 
```

Comment:
how is this related to Python 3 support?

Reply:
That needs to be explored, as it is very likely that it is exposing something which doesnt work on Python 3 and isnt covered by the test suite yet.